astropy/modeling/polynomial.py

```diff
@@ -381,10 +381,11 @@ def fit_deriv(self, x, *params):
         x = np.array(x, dtype=np.float, copy=False, ndmin=1)
         v = np.empty((self.degree + 1,) + x.shape, dtype=x.dtype)
         v[0] = x * 0 + 1
```

Comment:
While you are at it, just write
```
v[0] = 1
```
The `x * 0` part must be a leftover where broadcasting was needed or so.